CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   to find and add subdomains to the Sites Tree.
 - passive/JavaDisclosure.js - Passive scan for Java error messages leaks
 - httpsender/RsaEncryptPayloadForZap.py - A script that encrypts requests using RSA
+- selenium/Fill OTP in MFA.js - A script that fills the OTP in MFA
```

Comment:
Please take a look at https://github.com/zaproxy/community-scripts/blob/main/CONTRIBUTING.md

Reply:
Ok I'll fix the name; Actually I took inspiration from the naming convention followed in the scripts already present in repo i.e. https://github.com/zaproxy/community-scripts/blob/main/selenium/Selenium%20Juice%20Shop.js

Reply:
Ya we will have to crack down on that 🙂

Reply:
@kingthorin May I create an issue for that in main repo?

Reply:
https://github.com/zaproxy/community-scripts/pull/438#discussion_r1516227144

Not sure there's the need to rename old scripts, which would break users' workflows.

https://github.com/zaproxy/community-scripts/pull/438#discussion_r1516229631

The issue would be in this repo: https://github.com/zaproxy/community-scripts/issues